lib/new_super_codebreaker_2021.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require "yaml"
+require_relative "new_super_codebreaker_2021/version"
+require_relative "validate"
+require_relative "show_content"
+require_relative "db_methods"
+require_relative "user"
+
+module NewSuperCodebreaker2021
+  class Error < StandardError; end
+
+  class Game
+    include Validate
+    include ShowContent
+    include DBMethods
+    def chose_com(command)
+      case command
+      when "start" then :start
+      when "rules" then :rules
+      when "stats" then :stats
+      when "exit" then :exit
+      else false
+      end
+    end
+
+    def take_name(input_name)
+      if input_name == "exit"
+        :exit
+      else
+        validate_name(input_name)
+      end
+    end
+
+    def chose_difficulty(difficulty)
+      case difficulty
+      when "easy" then :easy
+      when "medium" then :medium
+      when "hell" then :hell
+      when "exit" then :exit
```

Comment:
we can use `.to_sym` for all difficulties, without `switch`

Reply:
done